pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -512,7 +512,17 @@ func (l LibvirtWrapper) SetupLibvirt() (err error) {
 		defer util.CloseIOAndCheckErr(libvirdDConf, &err)
 
 		// see https://libvirt.org/kbase/debuglogs.html for details
-		_, err = libvirdDConf.WriteString("log_filters=\"3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 1:*\"\n")
+		const defaultLogFilters = "3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device " +
```

Comment:
Why are we changing the default?

Reply:
Currently libvirt logs are very very verbose. This makes it hard to perform actual debug since there is so much noise going on.

This PR's purpose is to raise a discussion on which logs to remove. As you can see in the PR description, I've done my best to break this apart so we can discuss what's important and what's not.

Reply:
I like the addition of the possibility to modify the filter. I can't speak about the change of default filter as I am not that familiar with debugging of libvirt&qemu. I personally feel `QEMU_MONITOR_SEND_MSG` might be helpful.

@vladikr @andreabolognani  Can you have a look here?

Reply:
I wouldn't change the default. It is proven to be useful. Instead, I would suggest a variable approach. 
Today we can adjust the verbosity of the virt-launcher log in the CR:
```
configuration:
    developerConfiguration:
      logVerbosity:
        virtLauncher: 2
```
verbosity 5 and above will enable the libvirt debug logs. 
We can introduce several filters that would gradually increase the verbosity.

Reply:
The ability to configure libvirt's log filters per-VM sounds very useful, so I'm in favor of that. I would probably change the name of the annotation to `libvirt-log-filters` though, just to make it obvious which logs are affected by the filters.

Filtering out messages coming from `util.threadjob` and `cpu.cpu` is pretty much a no brainer, to the point that I will probably suggest making the same change to the [libvirt kbase article](https://libvirt.org/kbase/debuglogs.html). `conf.domain_addr` I'm a bit more on the fence about, but it's probably fine to filter that out too.

The QEMU monitor bits can be really useful in some debugging scenarios, so I'm not convinced they should go. @vladikr's suggestion to add more filters as the log verbosity is increased sounds appealing. So perhaps we could have your new settings as the base, your new settings + the QEMU bits when the log verbosity is increased by one, and the ability to set filters arbitrarily for cases that are not covered by the two built-in settings. Would that make sense?

Reply:
Thanks everyone!

What I've done:
* @vladikr: I've went with your approach. Now verbosity levels 5 to 8 gradually increase libvirt log entries. Verbosity level 8 is the same as verbosity level 5 before this PR.
* @andreabolognani: Further to previous bullet, now level 5 filters out everything I've filtered before your comments, level 6 includes `qemu.qemu_monitor_json` and `conf.domain_addr` categories, level 7 includes `qemu.qemu_monitor_json 3:conf.domain_addr` and level 8 includes `util.threadjob` and `cpu.cpu`. Thanks!
* @xpivarc: defined annotation in the right place. Thanks!

Reply:
WDYT? @vladikr @xpivarc @andreabolognani

Reply:
@iholder-redhat looks great to me!